README.md

```diff
@@ -20,12 +20,12 @@ The upcoming 2.0 version has drastically changed API. All available methods can
 
 # Use cases
 
-First of all, require the library. This is a UMD Module, so it will work for AMD, Typescript and CommonJS module systems.
+First of all, require the library. This is a UMD Module, so it will work for AMD, TypeScript and CommonJS module systems.
 
 ```javascript
 const Bowser = require("bowser"); // CommonJS
 
-import * as Bowser from "bowser" // Typescript
```

Comment:
I'm not sure if this one is relevant since, I guess, this `import * as Bowser ...` was used on purpose by @alexandercerutti when he was working on #277.

Reply:
Oh man... These typings are revealing to be a source of troubles...

That's weird. I tested without * if I remember correctly and it wasn't working... Tomorrow I'll check and let you know better.
Maybe Typescript import and ES6 import work differently.

Reply:
I've tried it locally and it said, that there is no default export from browser module, that's why it wouldn't work without an asterisk. But, I've tried to replace `export = Bowser;` to `export default Bowser;` and it seems working for me (and actually would be more correct definition, since it definitely exports a class, not a set of objects), but not sure if it will work for you, so, check it out, please :)

Reply:
That is weird, I installed today v2.0 in my project and it worked...

In the previous PR, we saw that exporting default was not working as intended...